src/transform/ThemeTransform.js

```diff
@@ -41,14 +62,18 @@ const setTheme = (document, theme) => {
  * @return {void}
  */
 const classifyElements = element => {
-  Polyfill.querySelectorAll(element, 'img').forEach(image => {
-    if (!ElementUtilities.closestInlineStyle(image, 'background')) {
+  Polyfill.querySelectorAll(element, 'img')
+    .filter(imageNeedsWhiteBackground)
+    .forEach(image => {
       image.classList.add(CONSTRAINT.IMAGE_NO_BACKGROUND)
-    }
-    if (!ElementUtilities.isNestedInTable(image)) {
-      image.classList.add(CONSTRAINT.IMAGE_NONTABULAR)
-    }
-  })
+    })
+  /* en > Away colours > 793128975 */
+  /* en > Manchester United F.C. > 793244653 */
+  /* en > Pantone > 792312384 */
+  Polyfill.querySelectorAll(element, 'div.color_swatch div, div[style*="position: absolute"]')
```

Comment:
I don't think I would have taken this approach to classify elements that we _don't_ want styled. After some consideration, I like it and believe that identifying exceptions to generally be superior to classifying everything else instead. The reason is that, in general, if I was writing a given article, I'd want the whole thing to be styled by default and to manually, where needed, what shouldn't be. The only unfortunate side-effect is that we have to use `:not()` but this is already how math formulas are handled. Good work!